### ISSUE TITLE
GTA III/Vice City/San Andreas: Widescreen fix prevents emergency vehicles from spawning

### ISSUE THREAD

**Issue by CookiePLMonster** [PCSX2/pcsx2_patches#454]
See https://github.com/ThirteenAG/WidescreenFixesPack/issues/682. The current widescreen fix doesn't seem to patch the constant used by the emergency vehicle spawns, so they don't spawn. In my 100% playthrough of Vice City I never saw a fire truck or an ambulance, and I believe this is the reason.

**Comment by CookiePLMonster** [comment]
San Andreas might be affected too - the issue exists there too, but I don't know if the current widescreen fix accounts for it or not.

**Comment by CookiePLMonster** [comment]
PC widescreen fixes patch this constant since forever, so it's a matter of porting that to the PS2.

**Comment by CookiePLMonster** [comment]
@PeterDelta Has this ever been fixed in the US version of San Andreas (SLUS-20946)? #489 only has PAL versions, and when playing the NTSC-U version right now I am not getting ambulance spawns.

**Comment by MackdannyTwitch** [comment]
I've seen ambulance spawns in the past, yeah. Even confirmed with PeterDelta it worked. I can try again just to see if the recent changes didn't break anything

**Comment by MackdannyTwitch** [comment]
Could it be due to how small the areas are?

**Comment by MackdannyTwitch** [comment]
Oh, i triggered poloce response in Tierra Robada, but that may not be what you're referring to.

**Comment by CookiePLMonster** [comment]
Yeah, this is about paramedics and firefighters. Police obviously do react everywhere on the map and they have a countryside and a desert cop model.

**Comment by mohammedelmasariy** [comment]
Ah guys i exrite the files on desktop and all icons disappeared?
